### DISCUSSION TITLE
Why default memory size is 64Mb?

### DISCUSSION THREAD

**Discussion by ook** [memcached/memcached#1167] [Q&A] [answered]
Hello,

Testing Memcached to act as cache for a dynamic website (well… obviously) I got surprised to understand the default memory limit for memcached (out of the box) is 64Mb. To be honest, beside IoT, I know no appliance running less than Gb of RAM: is that a low default value due to the long story of memcached or am I missing something about it and memcached is *meant* to use a very low quantity of RAM?

Thank you for your pointers

**Comment by dormando** [comment, marked as answer]
64m is basically the minimum and has been since the software was initially released many years ago. It's up to the user to understand how much RAM they want to dedicate to memcached and change that single parameter (`-m`). Keeping it low helps people from being surprised if they start it in an area without much memory available (embedded, CI systems, overcrowded servers, etc). So I don't see any reason to raise the default.

I've seen terabyte sized instances. Just give it however much RAM you feel like budgeting and hopefully that's enough to help offload your database or storage.

**Comment by ook** [comment]
Thank you for your quick answer, @dormando . All is in 👍🏻
